atlasdb-rocksdb-tests/build.gradle

```diff
@@ -4,4 +4,5 @@ apply from: "../gradle/shared.gradle"
 dependencies {
   testCompile project(":atlasdb-rocksdb")
   testCompile project(":atlasdb-tests-shared")
+  testCompile group: 'com.palantir.remoting1', name: 'tracing'
```

Comment:
This one is a bit strange... don't really mind though, as these tests (and the module) should be going away soon.